hicommonwealth/commonwealth
# user survey growl

## Link to Issue
Closes: #7428

## Description of Changes
-built user survey growl

## "How We Fixed It"


## Test Plan
- go to my branch and go to anywhere on the platform
- click the survey button and discord link and confirm they bring you to the correct places
- click the X and confirm that the growl closes
- refresh page and confirm that growl is back
- click on checkbox then the X and confirm that the growl dissappears
- refresh page and confirm that growl is no longer visible
